src/sass/_shame.scss

```diff
@@ -138,3 +138,54 @@ body .row {
 .csp-inline-patch-logout {
   padding: 2em 0;
 }
+
+// /education/apply/ uses this expander button for navigation
+// Where else can I put this? Also, it could use a better name.
+.expander-button {
+  &::after {
+    background-image: url(/img/arrow-down-white.svg);
+    background-position: right bottom;
+    background-repeat: no-repeat;
+    background-size: 0.6em auto;
+    display: inline-block;
+    margin-left: 0.5em;
+    margin-right: -0.3em;
+    width: 1em;
+    height: 1em;
+    content: ''; // Add content for accessibility purposes?
+    text-indent: -9999em;
+  }
+}
+
+// Yuck...
+#apply-expander-content {
+  max-height: 19em;
```

Comment:
Yeah, it has trouble animating from `max-height: 0em;` to `max-height: auto` or `initial` or anything not a number. I've noticed trouble with animation in the dev tools. It'd work without that line, but it won't animate.

Unless I'm missing something. I did try it without that in there first, though.